gatsby-node.js

```diff
@@ -239,6 +249,19 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     component: EventsTemplate
   });
 
+  // Integrations List
+  const allIntegrations = res.data.allIntegrations.nodes;
+
+  allIntegrations.forEach((integration) => {
+    createPage({
+      path: `/service-mesh-management/meshery/integrations/${integration.slug}`,
+      component: integrationTemplate,
+      context: {
+        slug: integration.slug,
```

Comment:
After the above change, it will become, `integration.fields.slug`

Reply:
And, remove this definition and update the one in the bottom (duplicate of this)

Reply:
done.